podcasts/Whats New/WhatsNew.swift

```diff
@@ -4,9 +4,11 @@ import SwiftUI
 class WhatsNew {
     struct Announcement {
         let version: Double
-        let image: String
+        let header: () -> AnyView
```

Comment:
Does this need to be a closure?

```suggestion
        let header: AnyView
```

Reply:
I thought about being a closure because then the view will only be returned if the closure is called. My concern is that if the property is a view itself, in the future all views might be loaded to the memory — even if we didn't need them.

Reply:
Makes sense. 👍